dummyserver/handlers.py

```diff
@@ -159,6 +161,18 @@ def redirect(self, request):
         headers = [('Location', target)]
         return Response(status='303 See Other', headers=headers)
 
+    def multi_redirect(self, request):
+        "Performs a redirect chain based on ``redirect_codes``"
+        codes = request.params.get('redirect_codes', '200').decode('utf-8')
+        codes = codes.split(',')
+        head, tail = codes[0], codes[1:]
```

Comment:
That also removes the need for a `join` later on, on line 173: the string will just remain intact.